### PR TITLE
[DAT-1455] - fix CI failures

### PR DESCRIPTION
**Pull request by ms-fcabrera**
[Jira Ticket](https://makingsense.atlassian.net/browse/DAT-1455)

Fix warnings in bash files to pass new CI requirements

We had a difference in the scope of prettier between running it locally and running it through the github actions. We decided to apply prettier to all files as it is done in the CI

### CODE REVIEW

File: build-n-publish-release-w-docker.sh [hunk 1]
```diff
@@ -41,28 +36,28 @@ pkgVersion=$versionPatch
...
+echo versionPatch: "$versionPatch"
+echo versionFull: "$versionFull"
+echo pkgVersion: "$pkgVersion"
+echo pkgBuild: "$pkgBuild"
+echo pkgCommitId: "$pkgCommitId"
 
 # TODO: Remove these steps in favor of publishing to `fromdoppler` organization.
 for environment in ${environments}; do
-    echo Publishing ${environment}...
+    echo Publishing "${environment}"...
```
Comment (andresmoschini) [inline build-n-publish-release-w-docker.sh:48]: In my opinion in this way is better:

```suggestion
    echo Publishing "${environment}..."
```

File: publish-commit-image-to-dockerhub.sh [hunk 2]
```diff
@@ -81,40 +81,39 @@ fi
...
-docker tag local-doppler-webapp:${environment}-commit-${commit} ${imageName}:${preReleasePrefix}${environment}-${versionMayorMinorPatchPre}
-docker tag local-doppler-webapp:${environment}-commit-${commit} ${imageName}:${preReleasePrefix}${environment}-${versionFullForTag}
+docker tag local-doppler-webapp:"${environment}"-commit-"${commit}" "${imageName}":${preReleasePrefix}"${environment}"
+docker tag local-doppler-webapp:"${environment}"-commit-"${commit}" "${imageName}":${preReleasePrefix}"${environment}"-"${versionMayor}"
+docker tag local-doppler-webapp:"${environment}"-commit-"${commit}" "${imageName}":${preReleasePrefix}"${environment}"-"${versionMayorMinor}"
+docker tag local-doppler-webapp:"${environment}"-commit-"${commit}" "${imageName}":${preReleasePrefix}"${environment}"-"${versionMayorMinorPatch}"
+docker tag local-doppler-webapp:"${environment}"-commit-"${commit}" "${imageName}":${preReleasePrefix}"${environment}"-"${versionMayorMinorPatchPre}"
+docker tag local-doppler-webapp:"${environment}"-commit-"${commit}" "${imageName}":${preReleasePrefix}"${environment}"-"${versionFullForTag}"
```
Comment (andresmoschini) [inline publish-commit-image-to-dockerhub.sh:112]:
```suggestion
docker tag "local-doppler-webapp:${environment}-commit-${commit}" "${imageName}:${preReleasePrefix}${environment}"
docker tag "local-doppler-webapp:${environment}-commit-${commit}" "${imageName}:${preReleasePrefix}${environment}-${versionMayor}"
docker tag "local-doppler-webapp:${environment}-commit-${commit}" "${imageName}:${preReleasePrefix}${environment}-${versionMayorMinor}"
docker tag "local-doppler-webapp:${environment}-commit-${commit}" "${imageName}:${preReleasePrefix}${environment}-${versionMayorMinorPatch}"
docker tag "local-doppler-webapp:${environment}-commit-${commit}" "${imageName}:${preReleasePrefix}${environment}-${versionMayorMinorPatchPre}"
docker tag "local-doppler-webapp:${environment}-commit-${commit}" "${imageName}:${preReleasePrefix}${environment}-${versionFullForTag}"
```

File: publish-commit-image-to-dockerhub.sh [hunk 3]
```diff
@@ -81,40 +81,39 @@ fi
...
-docker push ${imageName}:${preReleasePrefix}${environment}-${versionMayorMinor}
-docker push ${imageName}:${preReleasePrefix}${environment}-${versionMayorMinorPatch}
-docker push ${imageName}:${preReleasePrefix}${environment}-${versionMayorMinorPatchPre}
-docker push ${imageName}:${preReleasePrefix}${environment}-${versionFullForTag}
+docker push "${imageName}":${preReleasePrefix}"${environment}"
+docker push "${imageName}":${preReleasePrefix}"${environment}"-"${versionMayor}"
+docker push "${imageName}":${preReleasePrefix}"${environment}"-"${versionMayorMinor}"
+docker push "${imageName}":${preReleasePrefix}"${environment}"-"${versionMayorMinorPatch}"
+docker push "${imageName}":${preReleasePrefix}"${environment}"-"${versionMayorMinorPatchPre}"
+docker push "${imageName}":${preReleasePrefix}"${environment}"-"${versionFullForTag}"
```
Comment (andresmoschini) [inline publish-commit-image-to-dockerhub.sh:119]:
```suggestion
docker push "${imageName}:${preReleasePrefix}${environment}"
docker push "${imageName}:${preReleasePrefix}${environment}-${versionMayor}"
docker push "${imageName}:${preReleasePrefix}${environment}-${versionMayorMinor}"
docker push "${imageName}:${preReleasePrefix}${environment}-${versionMayorMinorPatch}"
docker push "${imageName}:${preReleasePrefix}${environment}-${versionMayorMinorPatchPre}"
docker push "${imageName}:${preReleasePrefix}${environment}-${versionFullForTag}"
```

File: public/index.html [hunk 4]
```diff
@@ -91,7 +91,7 @@
         });
         window['doppler-session-mfe-configuration'] = {
           dopplerLegacyBaseUrl: '%REACT_APP_DOPPLER_LEGACY_URL%',
-          useDummies: '%NODE_ENV%' === 'development' || '%REACT_APP_IS_DEMO_ENABLED%' === 'true'
+          useDummies: false
```
[the remark this answers is not in the document]
Comment (ms-fcabrera) [reply public/index.html:94]: Thanks for reviewing, I missed this change

**Comment by andresmoschini** [comment]
🚀

**Comment by andresmoschini** [comment]
For the first error:

image

You can do the same as here: https://github.com/FromDoppler/doppler-custom-domain/pull/204

For the prettier error, I am not sure.

**Pull request merged**